src/pages/Filing/FilingApp/FilingWarnings.tsx

```diff
@@ -0,0 +1,197 @@
+import submitWarningsAccept from 'api/requests/submitWarningsVerified';
+import useSblAuth from 'api/useSblAuth';
+import classnames from 'classnames';
+import FormWrapper from 'components/FormWrapper';
+import {
+  Alert,
+  Button,
+  Checkbox,
+  Heading,
+  Icon,
+  List,
+  ListLink,
+  TextIntroduction,
+  WellContainer,
+} from 'design-system-react';
+import { useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import type { SubmissionResponse } from 'types/filingTypes';
+import { FileSubmissionState } from 'types/filingTypes';
+import useGetSubmissionLatest from 'utils/useGetSubmissionLatest';
+import useInstitutionDetails from 'utils/useInstitutionDetails';
+import { NavigationPrevious } from './FilingNavButtons';
+import { FilingSteps } from './FilingSteps';
+import InstitutionHeading from './InstitutionHeading';
+
+const isSubmissionAccepted = (submission?: SubmissionResponse): boolean => {
+  return submission?.state === FileSubmissionState.SUBMISSION_ACCEPTED;
+};
+
+function FilingWarnings(): JSX.Element {
+  const navigate = useNavigate();
+  const auth = useSblAuth();
+  const { lei, year } = useParams();
+  const [boxChecked, setBoxChecked] = useState(false);
+  const [formSubmitLoading, setFormSubmitLoading] = useState(false);
+  const [formSubmitError, setFormSubmitError] = useState(null);
+
+  const {
+    data: submission,
+    isLoading: isSubmissionLoading,
+    isError: errorSubmissionFetch,
+  } = useGetSubmissionLatest({ lei, filingPeriod: year });
+
+  const {
+    data: institution,
+    isLoading: isInstitutionLoading,
+    isError: errorInstitutionFetch,
+  } = useInstitutionDetails(lei);
+
+  const institutionName = isInstitutionLoading
+    ? 'Loading...'
+    : errorInstitutionFetch
+      ? ''
+      : institution.name;
```

Comment:
NIT: On a different PR, I talked with @shindigira about these nested ternaries being a little hard to read / maintain sometimes, and I used to have them [as a eslint rule](https://eslint.org/docs/latest/rules/no-nested-ternary) at other gigs.

You can keep this "one nested deep" ternary if you'd like, or you can refactor it out into a super verbose, but more readable/maintainable function with early returns like...

```
  const getIsInstitutionName = (): string => {
    if (isInstitutionLoading) {
      return 'Loading...';
    }
    if (errorInstitutionFetch) {
      return '';
    }
    return institution.name;
  };
  const institutionName = getIsInstitutionName();
  ```
  
You could do a switch statement too or an immediately invoked function expression, but I'm not a huge fan of either. Just a little style thing to think about maybe for future PRs. 👍

Reply:
Hey, it's Bill from The Future who's seen the next PR where [you delete the `InstitutionName` logic](https://github.com/cfpb/sbl-frontend/pull/446/files#diff-0c71d1696cc565a968a6a0cfc6fcbf8d5b7470ea27d6b61a07dcd6ba3f88fbc2L33-L38). In that PR, I think we do still have this same `institutionName` logic copied in [two](https://github.com/cfpb/sbl-frontend/blob/434-warnings-part2/src/pages/Filing/FilingApp/FilingErrors/index.tsx#L33) possible [places](https://github.com/cfpb/sbl-frontend/blob/434-warnings-part2/src/pages/Filing/FilingApp/FileSubmission.tsx#L164), so maybe worth consolidating into an exported function? Quick ticket or fix maybe in the [next PR](https://github.com/cfpb/sbl-frontend/pull/446)?

Reply:
Yeah, I also didn't like my own code so was working to get rid of the pattern.  
 
I'll look at consolidating in the next PR to avoid having introducing any additional merge conflicts.